File: Android/app/src/main/java/com/hhp227/datemate/model/Post.kt
package com.hhp227.datemate.model

data class Post(
    var uid: String = "",
    var author: String = "",
    var title: String = "",
    var body: String = "",
    var starCount: Int = 0,
    var stars: Map<String, Boolean> = mapOf(),
    var key: String = ""
)

File: iOS/DateMate/Legacy/Model/Post.swift
//
//  Post.swift
//  DateMate
//
//  Created by 홍희표 on 2021/12/11.
//

import Foundation

struct Post: Identifiable {
    var id: String
    
    var author: String
    
    var title: String
    
    var body: String
    
    var starCount: Int
    
    var stars: [String: Bool]
    
    var key: String
}
